src/nvim/auevents.lua

```diff
@@ -101,6 +101,7 @@ return {
     'WinNew',                 -- when entering a new window
     'WinEnter',               -- after entering a window
     'WinLeave',               -- before leaving a window
+    'WinClosed',              -- after closing a window
```

Comment:
should be in alphabetical order. I see that `ChanOpen` and `WinNew` are out of order, will fix those.